buildres/mac/JabRef-dmg-setup.applescript

```diff
@@ -0,0 +1,39 @@
+tell application "Finder"
```

Comment:
Why do we need to include this script? Is it because of the image position? If yes, then please change the img instead. 

It's really ugly that one needs to copy-paste the installation scripts. In this way, we don't get the enhancments and fixes from the JDK. So if its possible, I would strongly advice not to copy paste them.

Reply:
It's actually not an installation script, it's just a macro script to set the background image and the icon position in the dmg. I figured out that I don't need to modify the variables. So we can always easily include upstream fixes.
It's explicity meant for overriding https://docs.oracle.com/en/java/javase/14/jpackage/override-jpackage-resources.html#GUID-62412365-821D-4736-A20D-70F2D47136D8

Reply:
So what did you actually changed then?

Reply:
<img width="686" alt="grafik" src="https://user-images.githubusercontent.com/320228/90321143-41554200-df47-11ea-93da-4c57373b0246.png">

I changed the window positions and the icon position, by moving the icon position a few pixes down and to the right.

Reply:
That's what I meant. You can also keep them in the standard location, and change the background image instead to adapt to the these locations.